app/assets/javascripts/discourse/app/routes/new-message.js

```diff
@@ -15,24 +14,11 @@ export default DiscourseRoute.extend({
     if (this.currentUser) {
       this.replaceWith("discovery.latest").then((e) => {
         if (params.username) {
-          // send a message to a user
-          User.findByUsername(encodeURIComponent(params.username))
-            .then((user) => {
-              if (user.can_send_private_message_to_user) {
-                next(() =>
-                  e.send("createNewMessageViaParams", {
-                    recipients: user.username,
-                    topicTitle: params.title,
-                    topicBody: params.body,
-                  })
-                );
-              } else {
-                bootbox.alert(
-                  I18n.t("composer.cant_send_pm", { username: user.username })
```

Comment:
Is `composer.cant_send_pm` used anywhere else or can it be deleted from the locale file?

Reply:
It's being used just below for group messages. 

https://github.com/discourse/discourse/blob/1472e47aae5bfdfb6fd9abfe89beb186c751f514/app/assets/javascripts/discourse/app/routes/new-message.js#L50